common-ui/src/main/java/org/simple/clinic/common/ui/components/SearchButton.kt

```diff
@@ -0,0 +1,48 @@
+package org.simple.clinic.common.ui.components
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.defaultMinSize
+import androidx.compose.foundation.layout.requiredWidth
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.ProvideTextStyle
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Search
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import org.simple.clinic.common.ui.theme.SimpleTheme
+
+@Composable
+fun SearchButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    content: @Composable RowScope.() -> Unit
+) {
+  androidx.compose.material.OutlinedButton(
```

Comment:
We have a custom `OutlinedButton` in `Button.kt`, any reason we didn't use that here? Is it just the text style?

Reply:
Actually let's replace the 2dp value, since it doesn't fit with the Material specs. Min is 4dp aka MaterialTheme.shapes.small.

But for text, we can just add that as a param for the `Button`s? Since we would have to use these buttons in other parts of the app where we will migrate screens to Compose. WDYT?